gnuboardProject/frontend/src/components/Profile/Profile.tsx

```
@@ -127,40 +245,84 @@ const Profile: React.FC = () => {
+      setImgPreview(URL.createObjectURL(e.target.files[0]));
+    }
+  };
+
+  const handlePhoneEditChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    let value = e.target.value.replace(/[^0-9]/g, '');
+    if (value.length < 4) {
+      setEditProfile(prev => ({ ...prev, phone: value }));
+    } else if (value.length < 8) {
+      setEditProfile(prev => ({ ...prev, phone: value.slice(0, 3) + '-' + value.slice(3) }));
+    } else {
+      setEditProfile(prev => ({ ...prev, phone: value.slice(0, 3) + '-' + value.slice(3, 7) + '-' + value.slice(7, 11) }));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)